scripts/rnnlm/lmrescore_pruned.sh

```diff
@@ -105,7 +97,6 @@ cp $indir/num_jobs $outdir
 
 $cmd JOB=1:$nj $outdir/log/rescorelm.JOB.log \
   lattice-lmrescore-kaldi-rnnlm-pruned --lm-scale=$weight $special_symbol_opts \
-    --lattice-compose-beam=$lattice_prune_beam \
```

Comment:
and revert this too, or explain why it's needed.    why did you remove the check above for word-list mismatch.  can you please go through this PR and make sure there are no more extraneous changes?